src/modules/twap/containers/TwapConfirmModal/TwapConfirmDetails.tsx

```diff
@@ -19,29 +22,38 @@ export function TwapConfirmDetails(_props: TwapConfirmDetailsProps) {
   return (
     <div>
       <span>TWAP order split in {numberOfPartsValue} equal parts</span>
+
+      {/* Sell amount per part */}
       <ReviewOrderModalAmountRow
         amount={inputPartAmount}
         fiatAmount={inputFiatAmount}
         tooltip="TODO: add tooltip"
         label={'Sell' + amountLabelSuffix}
       />
+
+      {/* Buy amount per part */}
       <ReviewOrderModalAmountRow
         amount={outputPartAmount}
         fiatAmount={outputFiatAmount}
         tooltip="TODO: add tooltip"
         label={'Buy' + amountLabelSuffix}
         isAmountAccurate={false}
       />
-      {/*TODO: implement logic for time fields*/}
-      <ConfirmDetailsItem tooltip="TODO: add tooltip" label={'Start time first' + partsSuffix} withArrow={false}>
-        {startTime}
-      </ConfirmDetailsItem>
+
+      {/* Start time */}
+      {/*<ConfirmDetailsItem tooltip="TODO: add tooltip" label={'Start time first' + partsSuffix} withArrow={false}>*/}
+      {/*  {startTime}*/}
+      {/*</ConfirmDetailsItem>*/}
```

Comment:
why removing it? we were going to make a proposal for the label, right?

Reply:
So, keep it static `Now`?

I'll do. I just had understood we'd hide it until it's decided how to display it